doc/proxy-clients.rst

```diff
@@ -136,6 +138,29 @@ cache invalidator optional in your code, you can (based on the environment)
 determine whether to inject the real client or the Noop client. The rest of your
 application then does not need to worry about the environment.
 
+.. _multiplexer client:
+
+Multiplexer Client
+~~~~~~~~~~~~~~~~~~
+
+The Multiplexer client allows multiple Proxy clients to be used during the standard
+cache invalidation, thus enabling multiple caches to be handled at once.
+It is useful when multiple caches exist in the environment and they need to be handled
+at the same time; the Multiplexer proxy client will forward the cache invalidation calls
+to all Proxy clients provided at creation time.::
+
+    use FOS\HttpCache\ProxyClient\MultiplexerClient;
+    use FOS\HttpCache\ProxyClient\Nginx;
+    use FOS\HttpCache\ProxyClient\Symfony;
+
+    $nginxClient = new Nginx($servers);
+    $symfonyClient = new Symfony([...]);
+    $client = new MultiplexerClient([$nginxClient, $symfonyClient]);
+
+.. note::
```

Comment:
instead of making this a note, lets make this a code comment above the `new MultiplexerClient` line.

and say: "// Expects an array of ProxyClientInterface in the constructor"